sycl/test-e2e/KernelAndProgram/kernel-bundle-merge-options.cpp

```diff
@@ -1,21 +1,24 @@
 // REQUIRES: gpu
-// RUN: %{build} -o %t.out %debug_option
+// RUN: %{build} -o %t.out -g
 // RUN: env SYCL_UR_TRACE=2 %{run} %t.out | FileCheck %s
 // UNSUPPORTED: hip
 
-// Debug option -g is not passed to device code compiler when CL-style driver
-// is used and /DEBUG options is passed.
-// XFAIL: cl_options
-// XFAIL-TRACKER: https://github.com/intel/llvm/issues/16309
+// Rather than %debug_option, we just use -g since it's supported everywhere
```

Comment:
Have you checked with our downstream `icx.exe`? Usually all those special substitutions are added for a reason, i.e. whatever we use on Windows tries to mimic `cl.exe` interface and therefore not every option is supported on Windows.

Reply:
Let me recheck.

Reply:
`-g` works everywhere.  And in xmain today `/Zi /EHsc`  work correctly with icx.exe (`/Zi` gets translated to `-g` when we call the driver )

So I'm not sure why this test started failing in the first place. Maybe `%debug_option` is being mis set?

@lbushi25  - you recently added the XFAIL to `kernel-bundle-merge-options.cpp`, when we are using the `cl_options`. Do you remember any details about that? Near as I can tell, this test should have been fine.

Reply:
my bad.  Thanks.